PushSharp.Apple/Exceptions.cs

```diff
@@ -70,7 +76,7 @@ public string ErrorStatusDescription
 			}
 		}
 
-		public override string ToString()
+		public string ToString_legacy()
```

Comment:
I'd just delete this method; we can always get it back from git history if ever needed.